### DISCUSSION TITLE
Error message in myst-js

### DISCUSSION THREAD

**Discussion by riziles** [executablebooks/meta#951] [Q&A] [answered]
I'm getting the following error message when trying to export a Markdown file to PDF with the MyST-JS CLI. It could be something I am doing wrong, but I'm not sure.

```
Could not convert from SVG to PNG: _build\temp\myst8OW6fe\images\icon-79dac6a8a0f132efcb58b75c659f0eae.svg - 
Error: Command failed: 
convert -density 600 _build\temp\myst8OW6fe\images\icon-79dac6a8a0f132efcb58b75c659f0eae.svg
_build\temp\myst8OW6fe\images\icon-79dac6a8a0f132efcb58b75c659f0eae.png
Invalid Parameter - 600
```

FWIW, I can't seem to export with `jupyter-book` either, but the error message there says I have no bounding box (I've tried many many different svg images, all result in the same error).

**Comment by rowanc1** [comment]
@fwkoch do you have any ideas here? My guess is that there is a different version of `imagemagik` that is expected.

**Reply by riziles** [reply]
@rowanc1 , I reported it as a bug here https://github.com/executablebooks/mystjs/issues/290 because I've been working on it for 2 weeks and still haven't found any workaround other than batch converting all my images to another format and then running a find-and-replace in all my MyST files. (FWIW, I've found the best results are produced by converting the images to PDF and then referencing those in the Markdown before compiling. Keeps the graphics scalable and the file sizes small.)

**Comment by riziles** [comment, marked as answer]
Resolved by installing Inkscape and adding to Windows PATH variable!
